Contribute two articles

SketchingDev opened the pull request:
This PR relates to a conversation about [my blog articles being featured on the Developer Center Blog](https://developer.genesys.cloud/forum/t/how-to-spy-on-web-service-data-actions/13884/2).

I'm happy to discuss any amendments you may like to the articles.

timsmithgenesys commented on source/2022-03-22-troubleshooting-data-actions/index.md:
```diff
@@ -0,0 +1,111 @@
+When you close ngrok it will also close the tunnel (which is unique each time), so you'll have to revert your
+Data Action's URL after inspecting the traffic.
+
+**This article on [troubleshooting Genesys Web Service Data Actions](https://sketchingdev.co.uk/blog/genesys-troubleshooting-data-actions.html)
```
This can go in an informational callout using this syntax:

```
:::info
This article on [troubleshooting Genesys Web Service Data Actions](https://sketchingdev.co.uk/blog/genesys-troubleshooting-data-actions.html)
was originally written for my personal blog, where I'll be writing more articles of this nature. If you have any
suggestions on how I can improve it, or my writing then please let me know [@SketchingDev](https://twitter.com/sketchingdev).
:::
```

SketchingDev replied:
3fa2178742440ad469b95cb37dc9d956875b4606

timsmithgenesys commented on source/2022-03-22-troubleshooting-data-actions/index.md:
```diff
@@ -0,0 +1,111 @@
+responses from a downstream API, breaking contracts/mappings. The following is how I sped up diagnosing problems by
+spying on the interaction between my Web Service Data Actions and downstream APIs.
+
+**WARNING: The tool used below should never be used to spy on Data Actions that are used by real customers. It
```
This can be shown in a warning callout using the following syntax:

```
:::warning
**WARNING:** The tool used below should never be used to spy on Data Actions that are used by real customers. It
terminates after a fixed period of time, has limited bandwidth and may leak secrets.
:::
```

timsmithgenesys commented on source/2022-03-23-automate-testing-genesys-web-messenger/index.md:
```diff
@@ -0,0 +1,128 @@
+
+## Writing automated tests
+
+The tool I've written uses [Web Messenger's guest API](https://developer.genesys.cloud/api/digital/webmessaging/websocketapi)
```
Because of the way this file gets included in the developer center, it's best not to include the domain in the link. Remove `https://developer.genesys.cloud` and just use the path starting with a forward slash as if this page were a part of the website instead of on github.

timsmithgenesys reviewed (approved):
The blog posts look great! Thanks for your submission!

timsmithgenesys commented:
@SketchingDev can you reply with as much of the information below as you'd like to include in your author bio? I'll also need a photo for the bio. It must be square, suggested resolution is 400x400px. Any kind of photo of yourself/avatar is fine as long as you hold the rights to the image and authorize Genesys to use it. Thanks!

```yaml
---
lucas.woodward:
  name: Lucas Woodward
  title: your job title
  company: your company name
  bio: a few sentences about yourself
  social:
    twitter: twitterhandle
    github: githubusername
    youtube: the channel ID of a youtube channel
```

timsmithgenesys commented on source/2022-03-22-troubleshooting-data-actions/index.md:
```diff
@@ -0,0 +1,111 @@
+date: 2022-03-22
+tags: data-actions, web service, troubleshooting
+author: lucas.woodward
+twitter: SketchingDev
```
twitter/github/linkedin should be removed from this section. leave author as-is though.

SketchingDev replied:
0c46455c0690abc48c522c47c5ee4d6e8ae126c4

timsmithgenesys commented on source/2022-03-23-automate-testing-genesys-web-messenger/index.md:
```diff
@@ -0,0 +1,128 @@
+date: 2022-03-23
+tags: web, messenger, testing, automation
+author: lucas.woodward
+twitter: SketchingDev
```
remove twitter, github, and linkedin here also

SketchingDev replied:
0c46455c0690abc48c522c47c5ee4d6e8ae126c4

SketchingDev commented:
@timsmithgenesys thank you for looking at the PR. I've addressed your comments and filled in the author information...

```
---
lucas.woodward:
  name: Lucas Woodward
  title: Software Engineer
  company: OVO Energy
  bio: Lucas is a full stack developer with over 15 years' experience who enjoys writing software for a living. Outside of work, he enjoys developing personal projects and attempting to draw.
  social:
    twitter: SketchingDev
    github: SketchingDev
    linkedin: lucas-woodward-264388108
```

You can use this photo of me:
![profile-photo](https://user-images.githubusercontent.com/31957045/160590431-8d5f5fb5-4954-44b6-8510-8b8e81c944c2.jpeg)

Many thanks

timsmithgenesys commented:
@SketchingDev blog posts have been published. Thanks again!

* https://developer.genesys.cloud/blog/2022-03-22-troubleshooting-data-actions/
* https://developer.genesys.cloud/blog/2022-03-23-automate-testing-genesys-web-messenger/
